tests/pytests/test_contains.py

```diff
@@ -128,12 +128,13 @@ def testSanity(env: Env):
         pl.execute()
 
     env.expect(config_cmd(), 'set', 'TIMEOUT', 1).ok()
-    run_command_on_all_shards(env, config_cmd(), 'SET', 'ON_TIMEOUT', 'RETURN')
+    env.expect(config_cmd(), 'set', 'ON_TIMEOUT', 'RETURN').ok()
```

Comment:
Shouldn't be the default?

Reply:
yes... I don't understand this test logic, but this is how it was before the change